model-optimizer/mo/main.py

```diff
@@ -400,3 +421,29 @@ def main(cli_parser: argparse.ArgumentParser, framework: str):
     telemetry.end_session()
     telemetry.force_shutdown(1.0)
     return 1
+
+
+def subprocess_main(framework=None):
```

Comment:
Add comment describing the function purpose and why we need sub-process.

Reply:
Done.